config/rector.php

```diff
@@ -11,9 +16,39 @@
             __DIR__ . '/../tests',
         ]
     )
-    ->withPhpSets()
-    ->withRules(
-        [
-            // AddVoidReturnTypeWhereNoReturnRector::class,
-        ]
-    );
+    ->withPhpVersion(PhpVersion::PHP_73)
+    ->withPhpSets(
+        true
+    )
+    ->withSets([
+        // Rector sets
+
+        LevelSetList::UP_TO_PHP_73,
+        // LevelSetList::UP_TO_PHP_74,
+        // LevelSetList::UP_TO_PHP_80,
+        // LevelSetList::UP_TO_PHP_81,
+        // LevelSetList::UP_TO_PHP_82,
+        // LevelSetList::UP_TO_PHP_83,
+
+        // SetList::CODE_QUALITY,
+        // SetList::CODING_STYLE,
+        // SetList::DEAD_CODE,
+        // SetList::EARLY_RETURN,
+        // SetList::INSTANCEOF,
+        // SetList::NAMING,
+        // SetList::PRIVATIZATION,
+        SetList::STRICT_BOOLEANS,
+        SetList::TYPE_DECLARATION,
+
+        // PHPUnit sets
+
+        PHPUnitSetList::PHPUNIT_90,
+        // PHPUnitSetList::PHPUNIT_100,
+        // PHPUnitSetList::PHPUNIT_CODE_QUALITY,
+    ])
+    ->withRules([
+        AddVoidReturnTypeWhereNoReturnRector::class,
+    ])
+    ->withImportNames(true, true, false)
```

Comment:
[This is all I found by way of documentation about `withImportNames`](https://getrector.com/documentation/import-names), and is insufficient to complete this review.

Reply:
This set of arguments means:

1. `import` classes instead of using the FQCM.
2. Also apply this to class names in doc blocks.
3. Do not `import` top level classes like `RuntimeException`